setup.py

```diff
@@ -15,6 +16,6 @@
     packages=[package_name],
     include_package_data=True,
     license=package["license"],
-    description=package["description"] if "description" in package else package_name,
```

Comment:
judging from the `setup.py` for Dash, we should [leave `description` in place](https://github.com/plotly/dash/blob/a9eb3434023880c229882dd2aacb7d501e7eb4d2/setup.py#L22) when we add `long_description`, as well as adding [`long_description_content_type`](https://github.com/plotly/dash/blob/a9eb3434023880c229882dd2aacb7d501e7eb4d2/setup.py#L27)

Reply:
Done!